libs/expo/betterangels/src/lib/screens/Client/Docs/UploadModal/index.tsx

```diff
@@ -0,0 +1,474 @@
+                  }}
+                >
+                  {!!docs.BirthCertificate && (
+                    <CheckIcon size="sm" color={Colors.WHITE} />
+                  )}
+                </View>
+
+                <Button
+                  disabled={!!docs.BirthCertificate}
+                  containerStyle={{ flex: 1 }}
+                  onPress={() => setTab('BirthCertificate')}
+                  height="md"
+                  align="flex-start"
+                  weight="regular"
+                  size="full"
+                  variant="secondary"
+                  title="Birth Certificate"
+                  accessibilityHint="goes to Birth Certificate upload"
+                />
+              </View>
+              <View
+                style={{
+                  flexDirection: 'row',
+                  alignItems: 'center',
+                }}
+              >
+                <View
+                  style={{
+                    height: 20,
+                    width: 20,
+                    borderRadius: Radiuses.xxxl,
+                    backgroundColor: docs.SocialSecurityCard
+                      ? Colors.SUCCESS
+                      : Colors.NEUTRAL_LIGHT,
+                    marginRight: Spacings.xs,
+                    alignItems: 'center',
+                    justifyContent: 'center',
+                  }}
+                >
+                  {!!docs.SocialSecurityCard && (
+                    <CheckIcon size="sm" color={Colors.WHITE} />
+                  )}
+                </View>
+
+                <Button
+                  disabled={!!docs.SocialSecurityCard}
+                  containerStyle={{ flex: 1 }}
+                  onPress={() => setTab('SocialSecurityCard')}
+                  height="md"
+                  weight="regular"
+                  align="flex-start"
+                  size="full"
+                  variant="secondary"
+                  title="SSN"
+                  accessibilityHint="goes to SSN upload"
+                />
+              </View>
+            </View>
+            <View style={{ gap: Spacings.xs, marginBottom: Spacings.lg }}>
+              <TextBold>Forms</TextBold>
+              <View
+                style={{
+                  flexDirection: 'row',
+                  alignItems: 'center',
+                }}
+              >
+                <View
+                  style={{
+                    height: 20,
+                    width: 20,
+                    borderRadius: Radiuses.xxxl,
+                    backgroundColor:
+                      docs.ConsentForm && docs.ConsentForm.length > 0
+                        ? Colors.SUCCESS
+                        : Colors.NEUTRAL_LIGHT,
+                    marginRight: Spacings.xs,
+                    alignItems: 'center',
+                    justifyContent: 'center',
+                  }}
+                >
+                  {!!docs.ConsentForm && docs.ConsentForm.length > 0 && (
```

Comment:
now that I think about it, this doesn't actually make sense. I can see wanting to do this for unique docs, like license and birth cert, but it doesn't make sense for forms, that aren't unique. see: [slack thread](https://betterangels-group.slack.com/archives/C05ECN9FLGL/p1725915484096409)

Reply:
@Davit-BetterAngels let's remove `disabled` for most of these buttons. The only buttons with `disabled` logic should be: license, birth cert, photoid, and social. Everything else should always be enabled.

Reply:
done